examples/internal-lb-cloud-run/main.tf

```diff
@@ -0,0 +1,147 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_compute_network" "default" {
+  name                    = "lb-internal-cloud-run"
+  auto_create_subnetworks = "false"
+  project                 = var.project_id
+}
+
+resource "google_compute_subnetwork" "default" {
+  name          = "lb-internal-subnet-cloud-run"
+  ip_cidr_range = "10.1.2.0/24"
+  network       = google_compute_network.default.id
+  region        = var.region
+  project       = var.project_id
+}
+
+resource "google_compute_subnetwork" "proxy_only" {
+  name          = "proxy-only-subnet-int"
+  ip_cidr_range = "10.129.0.0/23"
+  network       = google_compute_network.default.id
+  purpose       = "REGIONAL_MANAGED_PROXY"
+  region        = var.region
+  project       = var.project_id
+  role          = "ACTIVE"
+}
+
+module "backend-service" {
+  source                        = "GoogleCloudPlatform/cloud-run/google//modules/v2"
+  version                       = "~> 0.16.3"
+  project_id                    = var.project_id
+  location                      = var.region
+  service_name                  = "bs-2002"
+  containers                    = [{ "container_name" = "", "container_image" = "gcr.io/cloudrun/hello" }]
+  service_account_project_roles = ["roles/run.invoker"]
+  members                       = ["allUsers"]
+  ingress                       = "INGRESS_TRAFFIC_INTERNAL_ONLY"
+  cloud_run_deletion_protection = false
+  enable_prometheus_sidecar     = false
+}
+
+module "lb-http-backend" {
+  source  = "GoogleCloudPlatform/regional-lb-http/google//modules/backend"
+  version = "~> 0.4.0"
+
+  project_id            = var.project_id
+  region                = var.region
+  name                  = "backend-lb-int"
+  enable_cdn            = false
+  load_balancing_scheme = "INTERNAL_MANAGED"
+
+  serverless_neg_backends = [{ region : "us-central1", type : "cloud-run", service_name : module.backend-service.service_name }]
+}
+
+module "lb-http-frontend" {
+  source  = "GoogleCloudPlatform/regional-lb-http/google//modules/frontend"
+  version = "~> 0.4.0"
+
+  project_id            = var.project_id
+  region                = var.region
+  name                  = "frontend-lb-int"
+  url_map_input         = module.lb-http-backend.backend_service_info
+  network               = google_compute_network.default.name
+  subnetwork            = google_compute_subnetwork.default.name
+  load_balancing_scheme = "INTERNAL_MANAGED"
+
+  depends_on = [google_compute_subnetwork.proxy_only]
+}
+
+module "enable-vpc-access" {
+  source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  version = "~> 18.0"
+
+  project_id = var.project_id
+  activate_apis = [
+    "vpcaccess.googleapis.com"
+  ]
+}
+
```

Comment:
Let's move all API enablement to setup tf. This makes it less flaky.

Reply:
Done.